test/test_ops.py

```diff
@@ -436,7 +416,7 @@ def test_msroialign_repr(self):
         # Check integrity of object __repr__ attribute
         expected_string = (f"MultiScaleRoIAlign(featmap_names={fmap_names}, output_size={output_size}, "
                            f"sampling_ratio={sampling_ratio})")
-        self.assertEqual(t.__repr__(), expected_string)
+        assert t.__repr__() == expected_string
```

Comment:
Why not ?

```suggestion
        assert repr(t) == expected_string
```

Can be applied multiple times.